src/main/java/at/hannibal2/skyhanni/features/commands/PartyChatCommands.kt

```diff
@@ -7,23 +7,28 @@ import at.hannibal2.skyhanni.data.FriendApi
 import at.hannibal2.skyhanni.data.PartyApi
 import at.hannibal2.skyhanni.data.hypixel.chat.event.PartyChatEvent
 import at.hannibal2.skyhanni.events.chat.TabCompletionEvent
+import at.hannibal2.skyhanni.features.misc.CurrentPing
+import at.hannibal2.skyhanni.features.misc.TpsCounter
 import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
 import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.ConfigUtils.jumpToEditor
+import at.hannibal2.skyhanni.utils.DelayedRun
 import at.hannibal2.skyhanni.utils.HypixelCommands
 import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
 import at.hannibal2.skyhanni.utils.SimpleTimeMark
 import kotlin.time.Duration.Companion.seconds
 
 @SkyHanniModule
 object PartyChatCommands {
-
     private val config get() = SkyHanniMod.feature.misc.partyCommands
     private val storage get() = SkyHanniMod.feature.storage
 
     data class PartyChatCommand(
         val names: List<String>,
         val isEnabled: () -> Boolean,
         val requiresPartyLead: Boolean,
+        val worksOnSelf: Boolean,
```

Comment:
Make it default true so that we can save some lines below (same with requiresPartyLead, but this is maybe outside the scope)